<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerIGTLIONodeSelectorWidget</class>
 <widget class="QWidget" name="qSlicerIGTLIONodeSelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="qMRMLNodeComboBox" name="NodeSelector">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLLinearTransformNode</string>
       <string>vtkMRMLVolumeNode</string>
       <string>vtkMRMLModelNode</string>
       <string>vtkMRMLTextNode</string>
       <string>vtkMRMLMarkupsFiducialNode</string>
       <string>vtkMRMLIGTLTrackingDataBundleNode</string>
      </stringlist>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="removeEnabled">
      <bool>false</bool>
     </property>
     <property name="selectNodeUponCreation">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="AddNodeButton">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="RemoveNodeButton">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="SendButton">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
